processes/app_release.md

```diff
@@ -29,9 +29,10 @@
 
 13. `git tag vx.x.x`
 
-14. `git push origin stablex.x vx.x.x` (this means we push the commit + tags)
-15. Run `git push release stablex.x vx.x.x` to push it also to the release remote
-16. If the case is a minor or major version, you need to branch off this minor or major release: `git checkout -b  stablex.x` then do `git push origin stablex.x`
+14. `git push origin stablex.x vx.x.x` (this means we push the commit + tag)
+15. Run `git push stablex.x vx.x.x` to push it also to the release remote
```

Comment:
actually what we did was removing stablex.x, not release

so we did git push release v.13.0-beta1 instead of git push release main(aka stablex.x) v.13.0-beta1

Reply:
copy paste error :see_no_evil: